### PR TITLE
Fixes #169.

### LINKED ISSUE #169: Add setting to hide quotes
A user requested to be able to hide the quotes in the settings screen. Should be pretty straight forward to add a new shared preference and toggle the visibility!

### PR DESCRIPTION
**Pull request by peakvalleytech**
## :scroll: Description

Fixes issue #169.
## :green_heart: How did you test it?

Mannualy. Walked through all possible users flows for new feature.
## :camera_flash: Screenshots / GIFs

Screenshot_1615449045
Screenshot_1615449050
Screenshot_1615449146
Screenshot_1615449041

### CODE REVIEW

File: app/src/main/res/layout/testlayout.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (alisonthemonster) [inline app/src/main/res/layout/testlayout.xml]: I don't see this used anywhere, can it be removed?

File: ui/src/main/res/drawable/ic_tiny_flower.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="122dp"
+    android:width="180dp"
```
Comment (alisonthemonster) [inline ui/src/main/res/drawable/ic_tiny_flower.xml]: Would you mind reverting this here? After I merge this PR I'll go back and add the remaining translations and I'll make a new icon for this setting :)

**Comment by peakvalleytech** [comment]
@alisonthemonster Ok, i'll submit the changes soon.

**Comment by alisonthemonster** [comment]
Thanks for fixing those issues!

**Pull request merged**